deploy/olm-catalog/kubefed-operator/0.1.0/kubefed-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -402,85 +451,73 @@ spec:
         metadata:
           resourceVersion: ""
           selfLink: ""
+      ```
     See also the upstream [user guide](https://github.com/kubernetes-sigs/kubefed/blob/master/docs/cluster-registration.md#joining-clusters) for more information.
 
-    ### Unjoining Clusters
-
-    Use the `kubefedctl unjoin` command to remove a previously joined cluster.
-
-        kubefedctl unjoin mycluster \
-          --host-cluster-context mycluster \       # name of a KUBECONFIG context for the hosting cluster
-          --cluster-context mycluster \            # name of a KUBECONFIG context for the cluster to unjoin
-          --kubefed-namespace=<namespace> \        # namespace where kubefed is deployed
-          -v 2
-
-    This command deletes the `KubeFedCluster` and `Cluster` for the unjoining
-    cluster.
-
-        kubectl get kubefedclusters -n <namespace>
-        No resources found.
-    
     ### Enabling federation for new API types
 
-    kubefed allows you to spread any API type to target clusters. Use
-    `kubefedctl enable <API type>` for the API types you want to add federation
+    kubefed allows you to spread any API type to target clusters. Use `kubefedctl enable <API type>` for the API types you want to add federation
```

Comment:
s/kubefed allows you to spread any API type to/KubeFed allows you to manage any API type in/
s/add federation capability for/enable for management by KubeFed/